### PR TITLE
Rh gcp cert names #14492 keichwa

### PR DESCRIPTION
**Pull request by keichwa**
# Description

[client-cfg-rh-client.pdf](https://github.com/uyuni-project/uyuni-docs/files/6465238/client-cfg-rh-client.pdf)

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [x] Manager-4.2
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

Fixes https://github.com/SUSE/spacewalk/issues/14492

### CODE REVIEW

**Comment by keichwa** [comment]
@renner I think it would be good if a developer could check this PR.  But who?  I'll also ask Emiliano to take a look again.  In the above linked PDF, I changed the cert file names on p.6 of that file (AKA p. 80).

File: modules/client-configuration/pages/clients-rh-rhui.adoc [hunk 1]
```diff
@@ -73,14 +70,14 @@ The key has a file extension of [path]``.key``.
 . Copy your entitlement certificate and key from the client system, to a location that the {productname} Server can access:
 +
 ----
-cp /etc/pki/rhui/product/content-<version>.crt /<example>/entitlement/
-cp /etc/pki/rhui/content-<version>.key /<example>/entitlement/
+cp /etc/pki/rhui/product/content.crt /<example>/entitlement/
+cp /etc/pki/rhui/content.key /<example>/entitlement/
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-rh-rhui.adoc:74]:
```suggestion
cp /etc/pki/rhui/key.pem /<example>/entitlement/
```

**Review by keichwa — COMMENTED** [review]
per feedback

**Comment by emilianolangella** [comment]
Very appreciated Karl!

**Pull request merged**